distribution/docs/src/main/resources/_contents/_security-framework/developing-token-validators-contents.adoc

```diff
@@ -0,0 +1,9 @@
+
+==== Developing Token Validators
```

Comment:
It seems like we should have developing sections for the other parts of our security framework as well. In particular, `SecurityAssertion` and `PolicyExtension`.

There are also things like `PolicyPlugin`, which are `CatalogFramework` specific, but definitely security in nature. Where should we put the developer documentation for those?